src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { loadLocation } from '../../services/universal.discovery.service';
+
+import './css/content.creator.component.css';
+
+export default class ContentCreatorComponent extends Component {
+    constructor(props) {
```

Comment:
You can but I don't think you should.
`Class components should always call the base constructor with props.` from React doc: https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class

Reply:
This is recommendation, right. We can follow this rule. It's especially useful when you're extending your own component classes. In this case, when you extend base component class it's not that relevant. Because any kind of props won't affect the way the base class works.

Reply:
I know it will not cause any problems when extending `React.Component` for now, but if they have this in their doc they may require this at some point in the future.

Reply:
Ok, let's keep it then. From the currect perspective, it brings no value now, but it's a recommended by React doc.